app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
```

Comment:
It's great that you are testing the sum of the parts. However, this test only checks that the sum of the parts equals the original value. It would be good to also test that the length of the result equals the number of parts you want to split the integer into. This will ensure that the function works as expected.